Formula/create-dmg.rb

```diff
@@ -1,8 +1,8 @@
 class CreateDmg < Formula
   desc "Shell script to build fancy DMGs"
   homepage "https://github.com/andreyvit/create-dmg"
-  url "https://github.com/andreyvit/create-dmg/archive/v1.0.0.5.tar.gz"
-  sha256 "de76c8a7a1f4705720d61d39de7c87b7bc2acc7c35f6ec8d6d2dbdafcedc21b6"
+  url "https://t.keka.io/create-dmg.1.0.0.6.zip"
```

Comment:
No, please use upstream distribution.

Reply:
For sure! Just testing how to solve the previous Catalina test error without releasing it in upstream.